README.md

```diff
@@ -55,31 +61,49 @@ After you have installed GOV.UK Frontend the `@govuk-frontend` package will appe
 
 ## Import assets
 
-Import styles into your main Sass file.
+### Import styles
 
-To import all components, add the line:
+You need to import the GOV.UK Frontend styles into the main Sass file in your project.
+
+1. To import all components, add the below to your Sass file:
 ```CSS
 @import "@govuk-frontend/all/all";
 ```
 
-To import an individual component (for example, a button), add the line:
+2. To import an individual component (for example a button), add the below to your Sass file:
 ```CSS
 @import "@govuk-frontend/button/button";
 ```
 
+You should place the above code before your other Sass rules, or Sass imports, if you want to override
+GOV.UK Frontend with your own styles.
+
+In order to resolve the above `@import` paths(s) in your build, you should add `node_modules` to your [Sass include paths](https://github.com/sass/node-sass#includepaths) (in Ruby, they should be added to [assets paths](http://guides.rubyonrails.org/asset_pipeline.html#search-paths)).
```

Comment:
Could we do this the other way around.

By default tell people to import 'node_modules/@govuk-frontend/button/button' and give them the option to remove the initial node_modules with path.

We saw many people copy pasting without reading so this could be a good way to make this foolproof.

Reply:
This sounds like it might be a good approach ie. do `@import node-modules/@govuk-frontend/button` in scss , with the import paths route as optional. It makes the imports a bit more verbose but will allow people to get started quicker if this is what participants in user research struggled with. @alex-ju @36degrees @igloosi do you have any thoughts?

Reply:
Done!